Typesript nano types incorrect

## Bug Description

### 1. Steps to reproduce and the simplest code sample possible to demonstrate the issue
<!--
Outline the steps you take to make the problem happen.
Provide the simplest code sample you can, in context, that reproduces the issue.
-->

When attempting to use the default promise return type from `@cloudant/cloudant`, I get type errors when compiling, this is because of https://github.com/cloudant/nodejs-cloudant/blob/master/package.json#L26

code example of compilation failure:

```ts
import * as cloudant from '@cloudant/cloudant';

export function test(): Promise<any> {
    //cloudant object creation and db.use omitted
    return database.bulk({docs: [{foo: "bar"}]})
}
```
### 2. What you expected to happen

Complies successfully.

### 3. What actually happened

```
error TS2739: Type 'Request' is missing the following properties from type 'Promise<any[]>': then, catch, [Symbol.toStringTag]
```

## Environment details
- Version(s) that are affected by this issue.
    > 3.0.1
- Node.js version
    > 11.5.0

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,3 +1,6 @@
+# UNRELEASED
+- [FIXED] Remove unnecessary `@types/nano` dependancy.
+
 # 3.0.1 (2018-11-22)
 - [FIXED] Use named import for `request.CoreOptions` type.
 

--- a/package.json
+++ b/package.json
@@ -23,7 +23,6 @@
     "database"
   ],
   "dependencies": {
-    "@types/nano": "^6.4.5",
     "@types/request": "^2.47.0",
     "async": "2.1.2",
     "concat-stream": "^1.6.0",

